### PR TITLE
parent-doi: add link to record landing page

### PR DESCRIPTION
**Pull request by zzacharo**
:heart: Thank you for your contribution!

### Description

Please describe briefly your pull request.

### Checklist

Ticks in all boxes and 🟢 on all GitHub actions status checks are required to merge:

- [ ] I'm aware of the [code of conduct](https://inveniordm.docs.cern.ch/contribute/code-of-conduct/).
- [ ] I've created [logical separate commits](https://inveniordm.docs.cern.ch/develop/best-practices/commits/#commits) and followed the [commit message format](https://inveniordm.docs.cern.ch/develop/best-practices/commits/#commit-message).
- [ ] I've added relevant test cases.
- [ ] I've added relevant documentation.
- [ ] I've marked [translation strings](https://inveniordm.docs.cern.ch/develop/best-practices/i18n/) (for relevant code).
- [ ] I've followed the [CSS/JS](https://inveniordm.docs.cern.ch/develop/best-practices/css-js/) and [React](https://inveniordm.docs.cern.ch/develop/best-practices/react/) guidelines (for relevant code).
- [ ] I've followed the [web accessibility](https://inveniordm.docs.cern.ch/develop/best-practices/accessibility/) guidelines (for relevant code).
- [ ] I've followed the [user interface](https://inveniordm.docs.cern.ch/develop/best-practices/ui/) guidelines (for relevant code).
- [ ] I've identified the [copyright holder(s)](https://inveniordm.docs.cern.ch/contribute/copyright-policy/) and updated copyright headers for touched files (>15 lines contributions).
- [ ] I've NOT included third-party code (copy/pasted source code or new dependencies).

**Third-party code**

If you've added [third-party code (copy/pasted or new dependencies)](https://inveniordm.docs.cern.ch/develop/best-practices/commits/#third-party-codedependencies), please reach out to an [architect](https://github.com/orgs/inveniosoftware/teams/architects/members).

**Reminder**

By using GitHub, you have already agreed to the [GitHub’s Terms of Service](https://help.github.com/articles/github-terms-of-service/#6-contributions-under-repository-license) including that:

1. You license your contribution under the same terms as the current repository’s license.
2. You agree that you have the right to license your contribution under the current repository’s license.

### CODE REVIEW

**Review by kpsherva — APPROVED** [review]

File: invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/elements/list.overrides [hunk 1]
```diff
@@ -54,6 +54,18 @@
...
+    &.parent-doi:last-child {
+      padding-right: 0;
+      border-top: none;
+    }
+    &.parent-doi {
+      margin-top: .5rem;
+      word-wrap: break-word;
+    }
+  }
+  .ui.divided.list {
```
Comment (kpsherva) [inline invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/elements/list.overrides:67]: non needed?

File: invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/elements/list.overrides [hunk 2]
```diff
@@ -54,6 +54,18 @@
       word-wrap: break-word;
       font-size: 80%;
     }
+
+    &.parent-doi:last-child {
+      padding-right: 0;
```
Comment (kpsherva) [inline invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/elements/list.overrides:59]: pr-0 class can be used directly instead

File: invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/elements/list.overrides [hunk 3]
```diff
@@ -54,6 +54,18 @@
...
+      border-top: none;
+    }
+    &.parent-doi {
+      margin-top: .5rem;
+      word-wrap: break-word;
```
Comment (kpsherva) [inline invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/elements/list.overrides:64]: these already exist for class .doi, see list.overrides line 50, could be reused?

Comment (zzacharo) [reply invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/elements/list.overrides:64]: I tried it but display: block and font-size is not what we want for the parent doi....

File: invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/help/versioning.en.html [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+        </p>
+        <h5>
+          <p>Which DOI should I use in citations?</p>
+        </h5>
+        <p>
+          You should normally always use the DOI for the
+          <strong>specific version</strong> of your record in citations. This is to
+          ensure that other researchers can access the
+          <strong>exact</strong> research artefact you used for reproducibility. By
+          default, Zenodo uses the specific version to generate citations.
```
Comment (lnielsen) [inline invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/help/versioning.en.html:74]:
```suggestion
          default, we use the specific version to generate citations.
```

File: invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/help/versioning.en.html [hunk 5]
```diff
@@ -0,0 +1,195 @@
...
+        <p>
+          You can use the Concept DOI representing all versions in citations when it
+          is desirable to cite an evolving research artifact, without being specific
+          about the version.
+        </p>
+        <h5>
+          <p>Where does the Concept DOI resolve to?</p>
+        </h5>
+        <p>
+          Currently the Concept DOI resolves to the landing page of the latest
```
Comment (lnielsen) [inline invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/help/versioning.en.html:85]:
```suggestion
          The Concept DOI resolves to the landing page of the latest
```

File: invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/help/versioning.en.html [hunk 6]
```diff
@@ -0,0 +1,195 @@
...
+          version of your record. This is not fully correct, and in the future we
```
Comment (lnielsen) [inline invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/help/versioning.en.html:86]:
```suggestion
          version of your record. 
```

File: invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/help/versioning.en.html [hunk 7]
```diff
@@ -0,0 +1,195 @@
...
+          will change this to create a landing page specifically representing the
```
Comment (lnielsen) [inline invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/help/versioning.en.html:87]:
```suggestion

```

File: invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/help/versioning.en.html [hunk 8]
```diff
@@ -0,0 +1,195 @@
...
+          concept behind the record and all of its versions.
```
Comment (lnielsen) [inline invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/help/versioning.en.html:88]:
```suggestion
```

File: invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/help/versioning.en.html [hunk 9]
```diff
@@ -0,0 +1,195 @@
...
+        </p>
+        <h5>
+          <p>Do you support versioning for already existing records?</p>
```
Comment (lnielsen) [inline invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/help/versioning.en.html:91]: Delete this question

File: invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/help/versioning.en.html [hunk 10]
```diff
@@ -0,0 +1,195 @@
...
+        </p>
+      </div>
+      <div class="ui eight wide column">
+        <h5>
+          <p>
+            I only want to change the title of my upload, do I still get a new DOI?
+          </p>
+        </h5>
+        <p>
+          No, as before you can continue to edit the metadata of your upload without
```
Comment (lnielsen) [inline invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/help/versioning.en.html:123]:
```suggestion
          No, you can continue to edit the metadata of your upload without
```

File: invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/help/versioning.en.html [hunk 11]
```diff
@@ -0,0 +1,195 @@
...
+        <h5>
+          <p>Why don’t the DOIs have a version number suffix like “.v1”?</p>
+        </h5>
+        <p>
+          Including semantic information such as the version number in a DOI is bad
+          practice, because this information may change over time, while DOIs must
+          remain persistent and should not change.
+        </p>
+        <p>
+          Moreover, Zenodo DOI versioning is linear, which means that the Zenodo
```
Comment (lnielsen) [inline invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/help/versioning.en.html:137]:
```suggestion
          Moreover, DOI versioning is linear, which means that the
```

File: invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/help/versioning.en.html [hunk 12]
```diff
@@ -0,0 +1,195 @@
...
+          version number may in fact not be the real version number of the resource.
+          Take for instance software, where it is common practice to have dot
+          versions and make new releases in a non-linear order (e.g. first v1.0,
+          then v1.1, then v2.0, then v1.2).
+        </p>
+        <p>
+          The versioning suffix is also not a functionality of the DOI system, i.e.
+          adding .v2 to DOI will not resolve to version 2 of a resource for any DOI
```
Comment (lnielsen) [inline invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/help/versioning.en.html:145]:
```suggestion
          adding .v2 to a DOI will not resolve to version 2 of a resource for any DOI
```

File: invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/help/versioning.en.html [hunk 13]
```diff
@@ -0,0 +1,195 @@
...
+          in fact two versions of the same resource.
+        </p>
+        <p>
+          A better solution to this problem is to semantically link two DOIs in the
+          metadata of a DOI. This ensures that discovery systems have a
+          machine-readable way to discover that two DOIs are versions of the same
+          resource.
+        </p>
+        <p>
+          See also <a href="https://doi.org/10.5438/55E5-T5C0">Cool DOIs</a> blog
```
Comment (lnielsen) [inline invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/help/versioning.en.html:161]:
```suggestion
          See also <a href="https://doi.org/10.5438/55E5-T5C0">Cool DOIs</a> for further information.
```

File: invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/help/versioning.en.html [hunk 14]
```diff
@@ -0,0 +1,195 @@
...
+          post by Martin Fenner, DataCite Technical Director.
```
Comment (lnielsen) [inline invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/help/versioning.en.html:162]:
```suggestion

```

File: invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/help/versioning.en.html [hunk 15]
```diff
@@ -0,0 +1,195 @@
...
+        </p>
+
+        <h5>
+          <p>Why do you include “zenodo” in the DOI?</p>
```
Comment (lnielsen) [inline invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/help/versioning.en.html:166]: Remove this question

File: invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/help/versioning.en.html [hunk 16]
```diff
@@ -0,0 +1,195 @@
...
+            ></integer
+          >
+        </p>
+
+        <h5>
+          <p>Do you duplicate all the files for every new version of a record?</p>
+        </h5>
+        <p>
+          No, if you change a 10kb README file in 50GB dataset we do not duplicate
+          the entire 50GB dataset. Invenio v3, the underlying digital repository
```
Comment (lnielsen) [inline invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/help/versioning.en.html:188]:
```suggestion
          the entire 50GB dataset. InvenioRDM, the underlying technical software platform,
```

File: invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/help/versioning.en.html [hunk 17]
```diff
@@ -0,0 +1,195 @@
...
+          platform that powers Zenodo, efficiently handles the file storage so we
```
Comment (lnielsen) [inline invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/help/versioning.en.html:189]:
```suggestion
          efficiently handles the file storage so we
```

**Pull request merged**